pkg/statistics/handle/bootstrap.go

```diff
@@ -619,6 +638,15 @@ func (*Handle) initStatsBuckets4Chunk(cache statstypes.StatsCache, iter *chunk.I
 	}
 }
 
+func initStatsBucketsSQLGen(isPaging bool) string {
+	selectPrefix := "select /*+ ORDER_INDEX(mysql.stats_buckets,tbl) */ HIGH_PRIORITY table_id, is_index, hist_id, count, repeats, lower_bound, upper_bound, ndv from mysql.stats_buckets where is_index=1"
```

Comment:
This seems different from the old one. Could you please explain why we need `is_index=1` here?

Reply:
It's aligned with the non-concurrent version. This is to make the concurrent one and the non-concurrent one the same.

Reply:
Got it.